server/game/cards/locations/04/thegodseye.js

```diff
@@ -1,8 +1,12 @@
 const DrawCard = require('../../../drawcard.js');
 
 class TheGodsEye extends DrawCard {
-    setupCardAbilities() {
-        // TODO: Cannot be discarded.
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: this,
+            effect: ability.effects.cannotBeDiscarded(context => context.stage === 'effect')
```

Comment:
The card text indicates that it simply cannot be discarded from play, not that it cannot be discarded by card effects. Should this be just `cannotBeDiscarded()`?